Makefile

```diff
@@ -26,8 +26,8 @@ lab: venv
 test: venv
 	(. venv/bin/activate; \
 		python setup.py install; \
-		venv/bin/coverage run setup.py test;\
-		venv/bin/coverage report -m --fail-under 95;\
+		venv/bin/coverage run -m pytest test --ignore=test/integration/; \
```

Comment:
I was thinking having integration as a top level folder

Reply:
the tests rely on shared fixtures with other tests. I was thinking of pulling those fixtures from a common common module elsewhere but this seemed better

Reply:
That a good point. That being said could you also add a target which only runs the integration tests?

Reply:
will do